.tugboat/config.yml

```diff
@@ -36,33 +44,69 @@ services:
                 - docker-php-ext-install mysqli exif zip
 
-                # Install wp-cli
-                - curl -O https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
-                - chmod +x wp-cli.phar
-                - mv wp-cli.phar /usr/local/bin/wp
-
                 # Install WordPress.
                 - composer install --optimize-autoloader
 
                 # OPTION 2: WordPress Core lives in a subdirectory (in this example, 'docroot').
                 - ln -snf "${TUGBOAT_ROOT}/docroot" "${DOCROOT}"
 
                 # Set the wp-config.php and .htaccess files with the one you defined for Tugboat.
-                - rm -rf ${DOCROOT}/wp-config.php
-                - cp ${TUGBOAT_ROOT}/.tugboat/wp-config.tugboat.php ${DOCROOT}/wp-config.php
+                - cp ${TUGBOAT_ROOT}/.tugboat/wp-config.tugboat.php ${DOCROOT}/wp-config.local.php
                 - cp ${TUGBOAT_ROOT}/.tugboat/.htaccess ${DOCROOT}/.htaccess
 
-                # Update permalinks to remove the index.php.
-                - wp --allow-root --path="${DOCROOT}" option set permalink_structure /%postname%/
-                - wp --allow-root --path="${DOCROOT}" rewrite flush --hard
 
             # Phase 2 (update): Import files, database, or any other assets that your
             # website needs to run.
             # When you refresh a Tugboat Preview, the process starts here, skipping `init`.
             update:
-                # Import uploads
-                - tar -C /tmp -zxf uploads.tar.gz
-                - rsync -avz --delete /tmp/uploads/ "${TUGBOAT_ROOT}/docroot/wp-content/uploads/"
+                # Ensure our private SSH key is in place so that we can connect to remote services.
+                # Note: This assumes you have added your private key as a Tugboat environment variable for this project.
+                #       In this example, that environment variable is named "MY_PRODUCTION_SSH_KEY"
+                - echo "${MY_PRODUCTION_SSH_KEY}" > ~/.ssh/production_ssh_key
+                - chmod 600 ~/.ssh/production_ssh_key
+
+                ################################################
+                # Import WordPress database and files (uploads).
+                #
+                # This example shows creating a fresh installation.
+                # Below are more examples for importing database and files remotely.
+                - |
+                    wp --allow-root core install \
+                    --url="https://${TUGBOAT_SERVICE_URL_HOST}" \
+                    --title="Tugboat Demo" \
+                    --admin_user="tugboatadmin" \
+                    --admin_password="tugboatdemo" \
+                    --admin_email="tugboat@example.com" \
+                    --skip-email
+                - wp --allow-root theme activate kadence
+                - wp --allow-root option set permalink_structure /%postname%/
+                - wp --allow-root cache flush
```

Comment:
We're calling the `wp` CLI command here, but the installation commands were removed?

Reply:
I included `wp-cli-bundle` in composer, so now the site always has the WP cli available. The same way we include drush with composer for Drupal sites.

Reply:
I added more explaination of when WP-CLI is coming from, as well as an example for installing it directly on Tugboat.